addons/account/views/partner_view.xml

```diff
@@ -63,10 +64,23 @@
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
                     <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                    <field name="active" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+
+        <record id="view_account_position_search" model="ir.ui.view">
+            <field name="name">account.fiscal.position.search</field>
+            <field name="model">account.fiscal.position</field>
+            <field name="arch" type="xml">
+                <search string="Fiscal Position">
+                    <field name="name"/>
+                    <filter string="Archived" name="active" domain="[('active','=',False)]"/>
```

Comment:
Why here ? Can't you define the filter in the original search view instead of using inheritance ?